simulation/src/test/java/com/palantir/dialogue/core/Strategy.java

```diff
@@ -81,15 +84,29 @@ private static Channel pinUntilError(Simulation sim, Supplier<List<SimulationSer
     private static Channel roundRobin(Simulation sim, Supplier<List<SimulationServer>> channelSupplier) {
         return RefreshingChannelFactory.RefreshingChannel.create(channelSupplier, channels -> {
             List<LimitedChannel> limitedChannels =
-                    channels.stream().map(Strategy::noOpLimitedChannel).collect(Collectors.toList());
+                    channels.stream().map(UnlimitedChannel::new).collect(Collectors.toList());
             LimitedChannel limited = new RoundRobinChannel(limitedChannels);
-            limited =
-                    instrumentClient(limited, sim.taggedMetrics()); // will always be zero due to the noOpLimitedChannel
+            limited = instrumentClient(limited, sim.taggedMetrics()); // always be zero due to UnlimitedChannel
             Channel channel = new QueuedChannel(limited, DispatcherMetrics.of(sim.taggedMetrics()));
             return new RetryingChannel(channel, 4 /* ClientConfigurations.DEFAULT_MAX_NUM_RETRIES */);
         });
     }
 
+    private static Channel preferLowest(Simulation sim, Supplier<List<SimulationServer>> channelSupplier) {
+        Random pseudoRandom = new Random(21735712L);
+        return RefreshingChannelFactory.RefreshingChannel.create(channelSupplier, channels -> {
+            ImmutableList<LimitedChannel> limitedChannels = channels.stream()
+                    .map(c1 -> new ConcurrencyLimitedChannel(
+                            c1, () -> ConcurrencyLimitedChannel.createLimiter(sim.clock())))
```

Comment:
You can turn off ConcurrencyLimiters here (and use `UnlimitedChannel` instead) and you actually get faster performance in most conditions - the only case that's worse if all nodes are 500ing, we end up just blasting one node with a massive spike of traffic.

![all_nodes_500 PREFER_LOWEST](https://user-images.githubusercontent.com/3473798/74791693-22853400-52b3-11ea-911a-0f9e322e69b9.png)